en/docs/install-and-setup/setup-mi-server.md

```diff
@@ -0,0 +1,41 @@
+
+Go through the following steps to setup the Micro Integrator server for VSCode. 
```

Comment:
Let's say install Micro integrator server runtime and tooling by pointing to relevant links( runtime : https://mi.docs.wso2.com/en/latest/install-and-setup/install/installing-mi/, tooling: https://mi.docs.wso2.com/en/latest/develop/mi-for-vscode/install-wso2-mi-for-vscode/#install-micro-integrator-for-vs-code)